From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sat, 13 May 2017 16:16:01 +0200
Subject: hostapd: remove unused variable declarations in hostapd.sh

None of the variables in this "local" declaration are actually set in
wpa_supplicant_add_network().

Signed-off-by: Matthias Schiffer <mschiffer@universe-factory.net>

diff --git a/package/network/services/hostapd/files/hostapd.sh b/package/network/services/hostapd/files/hostapd.sh
index 988ebc7757fedfba3eba464a973824622c3af8c3..195b6ec45ac4def6484b322703be04433b209c7b 100644
--- a/package/network/services/hostapd/files/hostapd.sh
+++ b/package/network/services/hostapd/files/hostapd.sh
@@ -696,7 +696,6 @@ wpa_supplicant_add_network() {
 			;;
 		esac
 	}
-	local beacon_int brates mrate
 	[ -n "$bssid" ] && append network_data "bssid=$bssid" "$N$T"
 	[ -n "$beacon_int" ] && append network_data "beacon_int=$beacon_int" "$N$T"
 
